Manifests/ManagedPreferencesApplications/menu.nomad.login.ad.plist

```diff
@@ -856,6 +948,172 @@
+				<string>/Library/Security/SecurityAgentPlugins/NoMADLoginAD.bundle/Contents/Resources/NoMADFDEIcon.png</string>
+			</dict>
+			<dict>
+				<key>pfm_app_min</key>
+				<string>1.4+</string>
+				<key>pfm_default</key>
+				<false/>
+				<key>pfm_description</key>
+				<string>If enabled, NoMAD Login will only enable the first user that is eligable for a SecureToken, and delete the service account afterwards.</string>
+				<key>pfm_name</key>
+				<string>SecureTokenManagementOnlyEnableFirstUser</string>
+				<key>pfm_type</key>
+				<string>boolean</string>
+			</dict>					
+			<dict>
+				<key>pfm_app_min</key>
+				<string>1.4+</string>
+				<key>pfm_description_reference</key>
+				<string>Defines a custom Full Name for the SecureToken service account</string>
+				<key>pfm_name</key>
+				<string>SecureTokenManagementFullName</string>
+				<key>pfm_type</key>
+				<string>string</string>
+				<key>pfm_value_placeholder</key>
+				<string>NoMAD Login</string>
+			</dict>
+			<dict>
+				<key>pfm_app_min</key>
+				<string>1.4+</string>
+				<key>pfm_description_reference</key>
+				<string>Defines a custom UID for the SecureToken service account</string>
+				<key>pfm_name</key>
+				<string>SecureTokenManagementUID</string>
+				<key>pfm_type</key>
+				<string>string</string>
+				<key>pfm_value_placeholder</key>
+				<string>400</string>
+			</dict>			
+			<dict>
+				<key>pfm_app_min</key>
+				<string>1.4+</string>
+				<key>pfm_description_reference</key>
+				<string>Defines a custom password storage location for the SecureToken service account password</string>
+				<key>pfm_name</key>
+				<string>SecureTokenManagementPasswordLocation</string>
+				<key>pfm_type</key>
+				<string>string</string>
+				<key>pfm_value_placeholder</key>
+				<string>/var/db/.nomadLoginSecureTokenPassword</string>
+			</dict>					
+			<dict>
+				<key>pfm_app_min</key>
+				<string>1.4+</string>
+				<key>pfm_default</key>
+				<integer>16</integer>
+				<key>pfm_description</key>
+				<string>Defines a custom SecureToken service account password length.</string>
+				<key>pfm_name</key>
+				<string>SecureTokenManagementPasswordLength</string>
+				<key>pfm_range_list</key>
+				<array>
+					<integer>8</integer>
+					<integer>9</integer>
+					<integer>10</integer>
+					<integer>11</integer>
+					<integer>12</integer>
+					<integer>13</integer>
+					<integer>14</integer>
+					<integer>15</integer>
+					<integer>16</integer>
+					<integer>17</integer>
+					<integer>18</integer>
+					<integer>19</integer>
+					<integer>20</integer>
+					<integer>21</integer>
+					<integer>22</integer>
+					<integer>23</integer>
+					<integer>24</integer>
+					<integer>25</integer>
+					<integer>26</integer>
+					<integer>27</integer>
+					<integer>28</integer>
+					<integer>29</integer>
+					<integer>30</integer>
+					<integer>31</integer>
+					<integer>32</integer>
+				</array>
+				<key>pfm_range_list_titles</key>
```

Comment:
Since you're not defining a different string for the pfm_range_list values, the pfm_range_list_titles isn't needed